src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -95,6 +95,7 @@
 
     //PearlDiver
     protected int powThreads = Defaults.POW_THREADS;
+    protected String brokerIP = Defaults.BROKER_IP;
```

Comment:
If you are referring to IPv4 address, it is indeed a 32-bit integer. If you intended to mention `host`, you should revise the usage for these terms.

Reply:
Both of the `hostname` and the `IP address` expression should be acceptable.
The data type should use `String`.

The variable naming might be confusing.
I will change it to `brokerHostname` and update the description that both of the `hostname` and the `IP address` are acceptable.

Reply:
Done.